parser-typechecker/src/Unison/Runtime/IOSource.hs

```diff
@@ -303,7 +303,7 @@ unique[b28d929d0a73d2c18eac86341a3bb9399f8550c11b5f35eabb2751e6803ccc20] type
   IsPropagated = IsPropagated
 
 d1 Doc.++ d2 =
-  use Doc
+  use Doc2
```

Comment:
The original `use Doc` had no effect, although no `use` is needed here since there is only one `Join` constructor in the file.